docs/metrics-analysis-prometheus/collected-reported-metrics.md

```diff
@@ -80,24 +84,27 @@ for AWS (when creating an EC2 instance) and on the
 [Google Cloud website](https://cloud.google.com/compute/vm-instance-pricing)
 for GCP.
 
-Unfortunately, these values are not accurate as they lack several elements
-such as bandwidth or storage. As for storage costs, they can be found in
-the same places where the machine type cost is available. However, it is
-not possible to determine the bandwidth cost directly.
+We aim to obtain a result that is close to reality, within a range of
+approximately +/- 5%, for data visualization purposes.
+Key points to consider for retrieving cost information:
 
-To overcome this, we had to devise a workaround. We didn't necessarily
-need an exact cost for CI but rather a value close to reality (+/- 5%)
-for data visualization purposes.
+- RAM and CPU Costs : provided values for RAM and CPU expenses, can be found
+  in the Google Cloud documentation.
+- Storage Costs : provided values for storage expenses, can be found in the
+  Google Cloud documentation.
+- Bandwidth Cost: Directly determining the cost of bandwidth is not feasible.
 
-We analyzed previous invoices and calculated the additional cost generated
-by bandwidth, which amounted to approximately 30% for each month.
-Consequently, we were able to approximate the cost using the following formula:
+To arrive at an approximate cost, we conducted an analysis of previous invoices
+and calculated the additional expenses incurred due to bandwidth, which averaged
+around 30% per month. With this information, we were able to approximate the
+overall cost using the following formula:
 
 ```bash
 cost = (cost_per_flavor + cost_per_storage) * 130 / 100
```

Comment:
As the goal is to show a formula, would be good to see the bandwith mentioned as a variable and not a number. 

And review if it would be best to explicit mention that the cost is per minute.

Reply:
Done.
In my opinion, it is not necessary to specify "(per min)" as it is a formula and all variables have the same metric. In the sense that the formula remains true even if everything is expressed in hours. However, I did end up adding a note above.